tests/helpers/load_state_tests.py

```diff
@@ -323,14 +323,18 @@ def add_blocks_to_chain(
 
 
 def fetch_state_test_files(
-    test_dir: str, slow_test_list: Tuple[str, ...], load: BaseLoad
+    test_dir: str,
+    slow_test_list: Tuple[str, ...],
+    incorrect_tests_list: Tuple[str, ...],
+    load: BaseLoad,
 ) -> Generator[str, None, None]:
+    # TODO: provide a way to run slow tests
+    tests_to_ignore = slow_test_list + incorrect_tests_list
```

Comment:
Would using pytest markers be useful for this?

https://docs.pytest.org/en/7.1.x/example/markers.html

Reply:
done